core/src/lib/bsock_tcp.cc

```diff
@@ -2,7 +2,7 @@
    BAREOS® - Backup Archiving REcovery Open Sourced
 
    Copyright (C) 2007-2011 Free Software Foundation Europe e.V.
-   Copyright (C) 2013-2023 Bareos GmbH & Co. KG
+   Copyright (C) 2013-2024 Bareos GmbH & Co. KG
```

Comment:
revert

Reply:
that's going to happen when the fixups are squashed and bareos-check-sources is re-applied